internal/test/e2e/cluster_test.go

```diff
@@ -481,34 +479,36 @@ func TestObjectEntries(t *testing.T) {
 	}
 	for _, test := range tests {
 		// use the bus client
-		res, err := b.Object(context.Background(), api.DefaultBucketName, test.path, api.GetObjectOptions{
-			Prefix:  test.prefix,
-			SortBy:  test.sortBy,
-			SortDir: test.sortDir,
+		res, err := b.Objects(context.Background(), api.DefaultBucketName, test.path+test.prefix, api.ListObjectOptions{
+			Delimiter: "/",
```

Comment:
`api.DefaultDelimiter` | `api.DefaultObjectDelimiter`?

Reply:
I think keeping "" as the default is fine. There is only one test where we specify slash as the delimiter explicitly.